fied/datasets.py

```diff
@@ -48,7 +48,7 @@ def fetch_frs(combined=True):
             "NATIONAL_SIC_FILE.CSV",
             "NATIONAL_SUPP_INTEREST_FILE.CSV",
         ]
-    else:
+    else: 
```

Comment:
There is an extra space here that is better to remove it.

If you use Ruff to check and format the code, it will do this type of cleaning automatically for you.